subsys/metrics/src/main/resources/default-metrics.conf

```diff
@@ -0,0 +1,8 @@
+[metrics]
```

Comment:
I got little confuse, I need create file src/main/conf/conf.d/metrics.conf and default-metrics.conf.
Is it right? I should remove [metrics] header from these files ?